tools/circt-opt/CMakeLists.txt

```diff
@@ -1,9 +1,9 @@
 set(LLVM_LINK_COMPONENTS
   Support
-  )
+)
```

Comment:
Oof, looks like we're already inconsistent re:CMake formatting, thanks for touching it up.  This may be better if split into a separate PR.